src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -21,20 +24,34 @@
 $button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
 
 /**
- * Filter Modal Content
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters( 'tribe_tickets_ticket_block_submit_classes', [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: 1998f35cb83cce323dd67244227b70a9</sub>

Reply:
shut up bot.